server/CGameHandler.cpp

```diff
@@ -573,7 +573,7 @@ void CGameHandler::reinitScripting()
 #endif
 }
 
-void CGameHandler::init(StartInfo *si)
+void CGameHandler::init(StartInfo *si, Load::Progress *& progressTracking)
```

Comment:
Any way to avoid this weird "reference to pointer"?

Reply:
changed